server/models/db/timeslot.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+const Tables = require('../../constants/tables.json');
+
+// This method returns the timeslot records stored in the timeslot table 
+// The method will return all timeslots in the timeslot table  
+async function getPossibleTimeslots() {
+    const timeslotDetails = await db.any(`select * from ${Tables.timeslot} ;`);
+    return timeslotDetails;
```

Comment:
Hi Peter,
I would suggest mapping the response from the database into the individual `Timeslot` objects to ensure a consistent format is received in the front end. This would require importing the `Timeslot` data structure that currently exists in `models/data/Timeslot,js` and using `_.map(timeslotDetails, timeslot => new Timeslot(timeslot));`.

Reply:
Great catch Amy, this has been implemented and tested using Postman and `npm test` with 1005 coverage